psychopy/experiment/params.py

```diff
@@ -235,6 +235,15 @@ def __str__(self):
                 return valJS
             else:
                 return val
+        elif self.valType == 'color':
+            valid, val = self.dollarSyntax()
```

Comment:
Throw out unused `valid` like this: `_, val = self.dollarSyntax()` to keep LGTM happy.